src/Module.Server/Modes/Conquest/CrpgConquestServer.cs

```diff
@@ -343,6 +343,8 @@ private void TickFlags()
                 continue;
             }
 
+            float flagteamcountweight = 0;

```

Comment:
Could you rename `flagteamcountweight` to use camelCase and to something that is a bit more readable, maybe something like `flagCaptureWeight`?